jobs/integr8ly/poc-upgrade.yaml

```diff
@@ -0,0 +1,95 @@
+---
+
+- job:
+    name: poc-upgrade
+    project-type: pipeline
+    description: "Installs Integreatly remotely from Jenkins slave to QE Pony cluster."
```

Comment:
the description should be changed (I think it is probably the one used for poc-install)